unit.json

```diff
@@ -16,6 +16,21 @@
         }
     },
     "routes": {
+        "posthog": [
+            {
+                "match": {
+                    "uri": ["/healthz"]
```

Comment:
Just for my understanding, this is only for the billing pod?

Web deployment uses readyz / livez -> https://github.com/PostHog/charts/blob/f20ae5143fa5a5256b59e9e482a966ab7dce9b78/charts/posthog/templates/decide-deployment.yaml#L132-L153

Thinking about this, part of the problem might be our configuration settings? The liveness timeout is 2s, and readiness timeout is 5s: https://github.com/PostHog/charts/blob/f20ae5143fa5a5256b59e9e482a966ab7dce9b78/charts/posthog/values.yaml#L259-L269

I _think_ if we had a much larger liveness timeout, we'll have the pods continue running, and readiness will fail, which will mean no new traffic will come, but the pod will keep running. This _seems_ ideal for what we want?